config/prow/plugins.yaml

```diff
@@ -877,13 +877,11 @@ require_matching_label:
   repo: cluster-api
   issues: true
   prs: false
-  regexp: ^triage/accepted$
+  regexp: ^triage\/.+$
```

Comment:
Are you sure `/` has to be escaped? (because it was not escaped before)

Reply:
it should not make any difference, but ok to leave it as before

Reply:
Usually escaping is either required or it breaks something (or at least so I thought :))